controller/volume_controller.go

```diff
@@ -1028,24 +1028,38 @@ func (c *VolumeController) cleanupCorruptedOrStaleReplicas(v *longhorn.Volume, r
 	return nil
 }
 
-func (c *VolumeController) cleanupFailedToScheduledReplicas(v *longhorn.Volume, rs map[string]*longhorn.Replica) (err error) {
+func (c *VolumeController) cleanupFailedToScheduleReplicas(v *longhorn.Volume, rs map[string]*longhorn.Replica) (err error) {
```

Comment:
While you're at it, the name could change to cleanupFailedToScheduleReplica, since it used to do multiple replicas, and now it does the first such it finds.

Reply:
(Personally, I found the name hard to parse, until I mentally translated it as cleanupUnscheduledReplica.)

Reply:
TBH, this is a bit of an oversight on my part. When I read the original code, I thought it deleted one replica at a time as well. Reading it back again, I'm not sure why. I'm inclined to return to the original behavior and allow multiple deletions in a single reconcile. WDYT?

Reply:
I agree, I would stick with the original behavior unless there is a pressing reason to change, and I don't see one.

Reply:
Looks good.